### ISSUE TITLE
PyErr_Fetch always causes a nil pointer exception

### ISSUE THREAD

**Issue by mischief** [sbinet/go-python#37]
the code

```go
func PyErr_Fetch() (exc, val, tb *PyObject) {
	C.PyErr_Fetch(&exc.ptr, &val.ptr, &tb.ptr)
	return
}
```

will always cause a nil pointer exception because of the deref of the pointer return values. is this intentional?

**Comment by sbinet** [comment]
good catch.
fixed + added a test.

**Comment by mischief** [comment]
i think you maybe missed a file in your commit.

```
$ go test -v -run TestErrFetch
=== RUN   TestErrFetch
--- FAIL: TestErrFetch (0.00s)
        python_test.go:53: [tests/errfetch]: error running go-python test: chdir tests/errfetch: no such file or directory

FAIL
exit status 1
FAIL    github.com/sbinet/go-python     0.003s
```

**Comment by sbinet** [comment]
yep. fixed w/  2d6a486

**Comment by mischief** [comment]
thank you!
